src/numbers/extended_gcd.rs

```diff
@@ -0,0 +1,30 @@
+//! # Extended GCD
+
+/// See <https://nor-blog.codeberg.page/posts/2023-11-27-std-exchange-cpp/>
+///
+/// ext_gcd(a, b) returns (gcd, x, y) such that a * x + b * y == gcd
+/// if a and b are coprime, then x is the inverse of a (mod b)
+///
+/// # Example
+/// ```
+/// use programming_team_code_rust::numbers::extended_gcd::ext_gcd;
```

Comment:
why make the mod name and function name different? do you think it's more clear this way?

Reply:
no reason